package.json

```diff
@@ -63,7 +63,7 @@
   "scripts": {
     "serve": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --watchAll --coverage",
+    "test": "react-scripts test --watchAll",
```

Comment:
@Pranauv-Kumar1803 

Why have you modified the ```package.json``` file?

Reply:
It will cause all of the workflows to fail, since ```lcov.info``` will not be generated in this case.

Reply:
ok thank you

Reply:
Done @anshgoyalevil

Reply:
Thanks @Pranauv-Kumar1803